playbooks/common/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -52,13 +52,19 @@
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ openshift.common.admin_binary }} drain {{ openshift.node.nodename }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig --force --delete-local-data --ignore-daemonsets
+      {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }}
+      --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+      --force --delete-local-data --ignore-daemonsets
+      --timeout={{ openshift_upgrade_nodes_drain_timeout | default(0) }}s
     delegate_to: "{{ groups.oo_first_master.0 }}"
     when: l_docker_upgrade is defined and l_docker_upgrade | bool and inventory_hostname in groups.oo_nodes_to_upgrade
     register: l_docker_upgrade_drain_result
-    until: not l_docker_upgrade_drain_result | failed
-    retries: 60
-    delay: 60
+    until: not (l_docker_upgrade_drain_result is failed)
+    retries: "{{ 1 if ( openshift_upgrade_nodes_drain_timeout | default(0) | int ) == 0 else 0 }}"
```

Comment:
`openshift_upgrade_nodes_drain_timeout` is seconds, but `retries` is an amount of retries, so it should be `openshift_upgrade_nodes_drain_timeout / delay`.

Reply:
happy to take it in next PR but for now i think we are good since is just a cherry-pick

Reply:
Right, lets move all changes to a new PR - and start with master, as the tests are much more stable there